vector/build.gradle

```diff
@@ -229,7 +229,6 @@ android {
             buildConfigField "boolean", "LOW_PRIVACY_LOG_ENABLE", "false"
             // Set to true if you want to enable strict mode in debug
             buildConfigField "boolean", "ENABLE_STRICT_MODE_LOGS", "false"
-            buildConfigField "Boolean", "ENABLE_LIVE_LOCATION_SHARING", "true"
```

Comment:
we could have kept this but made the value `false`

Reply:
I am more in favor to remove it, for consistency with everything else we have in `DefaultVectorFeatures`

Reply:
for the future (might already be in your @bmarty config doc) it would be nice if we only have the config module as an entry point 

```
vector-config -> definitions and defaults
vector-features -> implementation of config subset with debug runtime overrides 
```

Reply:
I removed it since I wanted to reduce the number of configurations spot in the code. In the future we will have a lab setting for that feature.

Reply:
The lab setting will not replace the feature flag, it will be added, right?

Reply:
I don't know yet. I thought it would replace it but I didn't check how we previously handled such case.